django/contrib/admin/options.py

```diff
@@ -1769,7 +1768,7 @@ def history_view(self, request, object_id, extra_context=None):
             self.admin_site.each_context(request),
             title=_('Change history: %s') % obj,
             action_list=action_list,
-            module_name=capfirst(force_text(opts.verbose_name_plural)),
+            module_name=str(capfirst(opts.verbose_name_plural)),
```

Comment:
I'm surprised if `str()` is doing something here since `capfirst()` also has some `str()` calls.